test/parallel/test-process-uptime.js

```diff
@@ -20,18 +20,18 @@
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 'use strict';
-require('../common');
+const common = require('../common');
 const assert = require('assert');
 
 console.error(process.uptime());
-assert.ok(process.uptime() <= 2);
+// Add some wiggle room for different platforms.
+// Verify that the returned value is in seconds -
+// 15 seconds should be a good estimate.
+assert.ok(process.uptime() <= 15);
 
 const original = process.uptime();
 
 setTimeout(function() {
   const uptime = process.uptime();
-  // some wiggle room to account for timer
-  // granularity, processor speed, and scheduling
-  assert.ok(uptime >= original + 2);
-  assert.ok(uptime <= original + 3);
-}, 2000);
+  assert.ok(original < uptime);
+}, common.platformTimeout(100));
```

Comment:
We don't need to use `common.platformTimeout()` at all if all we're checking is `original < uptime`. For that matter, this can probably use a value more like `5` than `100`?